packages/fxa-content-server/tests/functional/settings_v2/display_name.js

```diff
@@ -8,30 +8,30 @@ const intern = require('intern').default;
 const { describe, it, beforeEach } = intern.getPlugin('interface.bdd');
 const selectors = require('../lib/selectors');
 const FunctionalHelpers = require('../lib/helpers');
+const FunctionalSettingsHelpers = require('..settings_v2/lib/helpers');
```

Comment:
Missing a `/` after `..`?

Reply:
Although I think the right path here is `../lib/helpers`.  Your text editor will probably able to you.